runtime/ast/expression_extractor.go

```diff
@@ -167,6 +171,26 @@ type ExpressionExtraction struct {
 	ExtractedExpressions []ExtractedExpression
 }
 
+// utility for expressions whose rewritten form is identical, i.e. nothing to rewrite
+func rewriteAsIs(expression Expression) ExpressionExtraction {
```

Comment:
See https://github.com/onflow/cadence/pull/2004#discussion_r982692582:

```suggestion
func rewriteExpressionAsIs(expression Expression) ExpressionExtraction {
```